settings.py.example

```diff
@@ -1,11 +1,14 @@
-DB_USER = ''
-DB_HOST = ''
-DB_PORT = ''
-DB_PASS = ''
-DB_NAME = ''
-DB_SCHEMA_AK = 'ak'
-DB_SCHEMA_SURVEY = 'survey_results'
-FUNCTION = 'survey_refresh_info'
-PAGE_ID = 1
-MINUTES = 60
-COLUMN_EXCLUDES = ''
+settings = {
```

Comment:
Updated this to a `dict` type to match the type of the settings returned by `get_settings()` in pywell, so we can use it the same way when we do `from test_settings import settings` in the test code. This change could have been avoided by renaming the `test_settings.py` file to `settings.py` and letting `get_settings()` pull the key/value pairs into a dictionary, but I thought it would be confusing that `settings.py` was formerly the real settings file and was now the test settings file.